Find a functionally equivalent code:com.android.internal.policy.impl.PhoneWindowManager.addStartingWindow:COMMENT
Method Modifier: public      internal    
Comment:/**
 * {@inheritDoc}
 */

@@ -58,64 +58,64 @@
         return view.getParent() != null ? view : null;
     } catch (WindowManager.BadTokenException e) {
         // ignore
-        Log.w(TAG, appToken + " already running, starting window not displayed");
+        Log.w(TAG, appToken + " already running, starting window not displayed. " + e.getMessage());
     } catch (RuntimeException e) {
         // don't crash if something else bad happens, for example a
         // failure loading resources because we are loading from an app

